src/modules/app/onBackNext.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import type { Dispatch, GetState, Bag } from 'redux-first-router/dist/flow-types'
+import { CATEGORIES_ROUTE } from './routes/categories'
+import { removeCategories } from '../endpoint/actions/remover'
+import { categoriesFetcher } from '../endpoint/fetchers'
+
+/**
+ * This handles the use of the back/ next button of the browser. We have to remove and refetch the categories in the old
+ * language, because the stopover of CategoriesRedirect, during which the refetch normally happens, is not executed again
+ * when using the back/ next button.
+ * @param dispatch
+ * @param getState
+ * @param bag
+ * @return {Promise<void>}
```

Comment:
You can remove this params returns or add docu :P

Reply:
removed :D